### PR TITLE
chore: add github issue template

### PR DESCRIPTION
**Pull request by rayrutjes**
Closes: #144

### CODE REVIEW

**Comment by rayrutjes** [comment]
For PR's I feel like it is a bit of an overhead for now.
Maybe we could introduce it the moment we see value in it.

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+**Do you want to request a *feature* or report a *bug*?**
+
+<!--
+  ***************************
+  If the current behavior is a bug, please provide all the steps to reproduce and a minimal
+  [CodePen](https://codepen.io/) example or a repository on GitHub that we can `npm install`
+  and `npm start`.
+
+  We have a simple online template for you to use as a base to explain your bug or issue:
+  https://codepen.io/rayrutjes/pen/BRgyGV
```
Comment (vvo) [inline .github/ISSUE_TEMPLATE.md]: It's easier to modify it (maybe), what we do on instantsearch.js is that we fork it on personnal account and change the link if needed. Not the best but still ok

**Comment by vvo** [comment]
> For PR's I feel like it is a bit of an overhead for now.
> Maybe we could introduce it the moment we see value in it.

👌

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (rayrutjes) [inline .github/ISSUE_TEMPLATE.md]: Yes that would be better.

**Pull request merged**